Pipfile

```diff
@@ -33,6 +33,7 @@ sentry-sdk = "==1.18.0"
 psycopg2 = "==2.9.5"
 kafka-python = "==2.0.2"
 psycopg2-binary = "==2.9.5"
+joserfc = "*"
```

Comment:
I would appreciate it if you could provide a specific version here 😄 thank you

Reply:
Done, thanks!